README.md

```diff
@@ -72,7 +73,8 @@ These will both result in the same JSON string written to STDOUT:
 {"ecs":{"version":"1.5.0"},"@timestamp":"2020-08-21T15:44:37.890Z","service":{"name":"service name"},"log":{"level":"error"},"message":"DB connection failed.","error":{"message":"Connection timed-out"}}
 ```
 
-Add log event specific information simply as attributes in a hash:
+#### Custom fields
```

Comment:
Hoping the header is enough of a hint that [most] fields are not populated magically, but open to feedback that this should be more explicit?